.github/workflows/labeller.yml

```diff
@@ -0,0 +1,22 @@
+name: community-labeller
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+
```

Comment:
nitpick-y but... is there a reason for whitespace here?

Reply:
No reason - resolved!